ui/src/telegrafs/components/TelegrafOutputOverlay.tsx

```diff
@@ -144,9 +158,15 @@ class TelegrafOutputOverlay extends PureComponent<Props> {
   }
 
   private handleDownloadConfig = () => {
+    const {server, org} = this.props
+    const bucket = this.currentBucket
     const config = transform(
       TELEGRAF_OUTPUT,
-      Object.assign({}, OUTPUT_DEFAULTS, {})
+      Object.assign({}, OUTPUT_DEFAULTS, {
```

Comment:
interesting that this isn't being destructured, might be worth considering when thinking about patterns @ebb-tide.

also, it looks like there's an extra space in indentation below, i think you'll need to run `prettier:fix`

Reply:
i dont understand